lib/x86/setup.c

```diff
@@ -2,13 +2,17 @@
  * Initialize machine setup information
  *
  * Copyright (C) 2017, Red Hat Inc, Andrew Jones <drjones@redhat.com>
+ * Copyright (C) 2021, UC San Diego, Zixuan Wang <zxwang42@gmail.com>
```

Comment:
This particular copyright is a little dicy, since `setup_tss()` was mostly developed at Google. I personally probably wouldn't have added to the new version. However, I think it's pretty subjective. So please feel free to leave it in if you disagree.

Reply:
See the other comment thread. I'm ok either way on whether or not to keep the new copyright. Thanks!

Reply:
I just updated all copyrights and commit tags with Google emails, just to make things easier. The Google email also represents me so I'm 100% ok with it, don't worry :-)